### PR TITLE
Improved wp_calculate_image_sizes() inline docblock

### PR DESCRIPTION
**Pull request by dilipbheda**
Trac ticket: https://core.trac.wordpress.org/ticket/59745

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/media.php [hunk 1]
```diff
@@ -1520,8 +1520,8 @@ function wp_get_attachment_image_sizes( $attachment_id, $size = 'medium', $image
  *
  * @param string|int[] $size          Image size. Accepts any registered image size name, or an array of
  *                                    width and height values in pixels (in that order).
- * @param string       $image_src     Optional. The URL to the image file. Default null.
- * @param array        $image_meta    Optional. The image meta data as returned by 'wp_get_attachment_metadata()'.
+ * @param string|null  $image_src     Optional. The URL to the image file. Default null.
+ * @param array|null   $image_meta    Optional. The image meta data as returned by 'wp_get_attachment_metadata()'.
```
Comment (mukeshpanchal27) [inline src/wp-includes/media.php:1524]:
```suggestion
 * @param array|null   $image_meta    Optional. The image meta data as returned by 'wp_get_attachment_metadata()'. Default null.
```

Add `Default null.` in document.

Comment (dilipbheda) [reply src/wp-includes/media.php:1524]: @mukeshpanchal27 `Default null.` already there
Ref: https://github.com/dilipbheda/wordpress-develop/blob/ticket/59745/src/wp-includes/media.php#L1524

Comment (dilipbheda) [reply src/wp-includes/media.php:1524]: @mukeshpanchal27 Let me know if any changes are required in this PR

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r57042](https://core.trac.wordpress.org/changeset/57042).

**Pull request closed without merging**